docs/smart-contracts/install/local-dev-node.md

```diff
@@ -0,0 +1,247 @@
+
+You can customize the operation of the development network using different command-line settings.
+For example, if you don't want to create a persistent volume, you can start the network by running the following command:
+
+```bash
+docker run --interactive --tty --publish 8080:8080 kadena/devnet:latest
+```
+
+You can stop the network at any time—and reset the blockchain state—by pressing Ctrl-c in the terminal. 
+After you stop the network, restart it using the previous command or one of the commands that follow.
+
+If you want to automatically remove the container when you stop it, add the `--rm` command-line option.
+For example:
+
+```shell
+docker run --rm --interactive --tty --publish 8080:8080 --volume kadena_devnet:/data --name devnet kadena/devnet
+```
+
+If you encounter an error where the version of Chainweb is invalid after a certain date (typically after a service update), run the following command to pull the latest version:
+
+```shell
+docker run --pull always --interactive --tty --publish 8080:8080 kadena/devnet:latest
+```
+
+If you can't run the Pact executable on your local computer, you can mount the `./pact` folder for the election project in the Docker container by running the following command:
+
+```bash
+docker run --interactive --tty \
+ --publish 8080:8080 \
+ --volume ./pact:/pact-cli:ro \
+ kadena/devnet:latest
+```
+
+### Connect Chainweaver to the development network
+
+By default, the Chainweaver wallet and IDE lets you connect to the Kadena test network and the Kadena main network. 
+However, as you start writing Pact modules, you'll want to test and deploy them on your local development network. Before you can do that, you need to configure Chainweaver to connect to the local host and port number running the development network.
+
+To add the development network to Chainweaver:
+
+1. Open or unlock Chainweaver.
+2. Click **Settings** in the Chainweaver navigation panel.
+3. Click **Network**.
+4. Under Edit Networks, type the network name **development**, then click **Create**.
+5. Expand the **development** network, then add the localhost as a node for this network by typing **127.0.0.1:8080**.
+
+   If the local computer is still running the development network Docker container, you should see the dot next to the node turn green.
+
+6. Click **Ok** to close the network settings.
+
+   After you click Ok, you can see **development** selected as your current network.
+   All actions you perform in Chainweaver are now executed on the local development network.
+
+## Install a standalone node
+
+If you want full control over node operations, you might want to install and manage `chainweb-node` as a standalone local node for development or as a node participating in the Kadena test or production public network.
+
+There are several options for setting up a Chainweb node in a physical or virtual environment.
+For example, you can install and run node binaries:
+
+- Directly on a physical Linux or macOS computer.
```

Comment:
```suggestion
- Directly on a physical Linux.
```

To my knowledge, currently ARM CPUs are not supported (which also includes pact)